backend/backend_main.py

```diff
@@ -100,11 +101,31 @@ def generate_answer(question, num_of_context=5)->str:
 
     qa_template = PromptTemplate(template)
     prompt = qa_template.format(context=context, question=question)
-    
-    messages = [
-        {"role": "user", "content": prompt},
-    ]
-    # output = pipe(messages, **TEXT_GENERATION_ARGS)
-    # answer = output[0]['generated_text']
-    return context[:200]
+    """ Get the response from the chatgpt 4 model """
+    api_key = os.getenv('OPENAI_API_KEY')
+    output = get_chatgpt_response(api_key, prompt)
+    answer = output["choices"][0]["message"]["content"]
+
+    return answer 
+
+
+
+import requests
+
+def get_chatgpt_response(api_key, prompt):
+    url = "https://api.openai.com/v1/chat/completions"
+    headers = {
+        "Content-Type": "application/json",
+        "Authorization": f"Bearer {api_key}"
+    }
+    data = {
+        "model": "gpt-4-turbo", 
+        "messages": [
+            {"role": "system", "content": "You are a helpful assistant."},
```

Comment:
Replace this with "You are his holiness the 14th Dalai Lama." and remove it from the prompt.